evm/src/cpu/kernel/asm/bignum/util.asm

```diff
@@ -1,13 +1,15 @@
-%macro memcpy_kernel_general
+%macro memcpy_current_general
     // stack: dst, src, len
-    %stack (dst, src, len) -> (0, @SEGMENT_KERNEL_GENERAL, dst, 0, @SEGMENT_KERNEL_GENERAL, src, len, %%after)
+    GET_CONTEXT
+    %stack (context, dst, src, len) -> (context, @SEGMENT_KERNEL_GENERAL, dst, 0, @SEGMENT_KERNEL_GENERAL, src, len, %%after)
     %jump(memcpy)
 %%after:
 %endmacro
 
-%macro clear_kernel_general
+%macro clear_current_general
     // stack: dst, len
-    %stack (dst, len) -> (0, @SEGMENT_KERNEL_GENERAL, dst, 0, len, %%after)
+    GET_CONTEXT
+    %stack (context, dst, len) -> (context, @SEGMENT_KERNEL_GENERAL, dst, 0, len, %%after)
```

Comment:
ditto

Reply:
This one is different, though: for tis `clear` macro, `0` is the value we're `memset`ing into the memory